gateway/modules/eventing/helpers.go

```diff
@@ -47,10 +47,14 @@ func (m *Module) transmitEvents(eventToken int, eventDocs []*model.EventDocument
 	}
 }
 
-func (m *Module) batchRequests(ctx context.Context, requests []*model.QueueEventRequest) error {
+func (m *Module) getBatchID() string {
+	return fmt.Sprintf("%s.%s", ksuid.New().String(), m.syncMan.GetNodeID())
```

Comment:
Let's use `--` as a delimiter to prevent conflicts which might happen in the future